### PR TITLE
Check if  sensu_available_checks is skippeded

### LINKED ISSUE #194: Deploy check plugins fatal error when running `--check`
When re-running the playbook in `--check` mode I am getting the following error:

```bash
TASK [external/sensu-ansible : Register available checks] ********************************************************************************************************************************************************************
skipping: [apps-test.redacted.com]
skipping: [sensu.redacted.com]

TASK [external/sensu-ansible : Deploy check plugins] *************************************************************************************************************************************************************************
fatal: [apps-test.redacted.com]: FAILED! => {"msg": "The conditional check 'sensu_available_checks is defined and item in sensu_available_checks.stdout_lines' failed. The error was: error while evaluating conditional (sensu_available_checks is defined and item in sensu_available_checks.stdout_lines): Unable to look up a name or access an attribute in template string ({% if sensu_available_checks is defined and item in sensu_available_checks.stdout_lines %} True {% else %} False {% endif %}).\nMake sure your variable name does not contain invalid characters like '-': argument of type 'StrictUndefined' is not iterable\n\nThe error appears to have been in '/Users/michaelporter/redacted/Ansible/roles/external/sensu-ansible/tasks/plugins.yml': line 39, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Deploy check plugins\n  ^ here\n"}
fatal: [sensu.redacted.com]: FAILED! => {"msg": "The conditional check 'sensu_available_checks is defined and item in sensu_available_checks.stdout_lines' failed. The error was: error while evaluating conditional (sensu_available_checks is defined and item in sensu_available_checks.stdout_lines): Unable to look up a name or access an attribute in template string ({% if sensu_available_checks is defined and item in sensu_available_checks.stdout_lines %} True {% else %} False {% endif %}).\nMake sure your variable name does not contain invalid characters like '-': argument of type 'StrictUndefined' is not iterable\n\nThe error appears to have been in '/Users/michaelporter/redacted/Ansible/roles/external/sensu-ansible/tasks/plugins.yml': line 39, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Deploy check plugins\n  ^ here\n"}
```

I added a debug to check the value:

```bash
TASK [external/sensu-ansible : debug] ****************************************************************************************************************************************************************************************
ok: [apps-test.redacted.com] => {
    "sensu_available_checks": {
        "changed": false,
        "failed": false,
        "msg": "skipped, running in check mode",
        "skipped": true
    }
}
ok: [sensu.redacted.com] => {
    "sensu_available_checks": {
        "changed": false,
        "failed": false,
        "msg": "skipped, running in check mode",
        "skipped": true
    }
}
```

### PR DESCRIPTION
**Pull request by jaredledvina**
I think this might resolve https://github.com/sensu/sensu-ansible/issues/194

### CODE REVIEW

**Comment by michaelpporter** [comment]
Looks good. It is also needed below on line 98

```yaml
- name: Deploy client definitions
  copy:
    src: "{{ static_data_store }}/sensu/client_definitions/{{ item }}/"
    dest: "{{ sensu_config_path }}/conf.d/{{ item | basename | regex_replace('.j2', '')}}"
    owner: "{{ sensu_user_name }}"
    group: "{{ sensu_group_name }}"
  when:
    - sensu_available_checks is defined
    - sensu_available_checks is not skipped
    - item in sensu_available_checks.stdout_lines
  with_flattened:
    - "{{ group_names }}"
  notify: restart sensu-client service

```

File: tasks/plugins.yml [hunk 1]
```diff
@@ -40,7 +40,10 @@
     mode: 0755
     owner: "{{ sensu_user_name }}"
     group: "{{ sensu_group_name }}"
-  when: "sensu_available_checks is defined and item in sensu_available_checks.stdout_lines"
+  when:
```
Comment (michaelpporter) [inline tasks/plugins.yml:43]: Also needed on `- name: Deploy client definitions`

Comment (jaredledvina) [reply tasks/plugins.yml:43]: Thanks, pushed up https://github.com/sensu/sensu-ansible/pull/195/commits/cee83c9d614dcdbbce59a19210adeccca636fe01

**Comment by michaelpporter** [comment]
Looks good. Made the changes locally this morning and tested. I think it is good to go.

**Pull request merged**